Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/DirectionIndicator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.MLAgentsExamples
+{
+    public class DirectionIndicator : MonoBehaviour
+    {
+        public Transform transformToFollow; //ex: hips or body
+        public Transform targetToLookAt; //target in the scene the indicator will point to
+        public float heightOffset;
+        private Vector3 m_StartingPos;
+        
+        void OnEnable()
+        {
+            m_StartingPos = transform.position;
+        }
+
+        void Update()
+        {
+            transform.position = new Vector3(transformToFollow.position.x, m_StartingPos.y + heightOffset, transformToFollow.position.z);
+            Vector3 m_WalkDir = targetToLookAt.position - transform.position;
```

Comment:
Since this is not a member variable, you do not need `m_` nor capitalization.

Reply:
ok will fix

Reply:
fixed